### DISCUSSION TITLE
如何修改.paddlenlp的路径？

### DISCUSSION THREAD

**Discussion by JoeLiaoCHN** [PaddlePaddle/PaddleNLP#309] [Q&A] [answered]
我在使用paddlehub下载“ernie-tiny”时，paddlehub会将ernie-tiny的部分文件下载到\.paddlenlp\models\ernie-tiny\下。
放在\.paddlenlp\models\ernie-tiny\路径下的文件名如下：

- dict.wordseg.pickle
- ernie_tiny.pdparams
- spm_cased_simp_sampled.model
- vocab.txt

由于我的用户名有中文字符，在使用下载好的文件时，会报OSError的错误。
我现在已经将paddlehub的“HUB_HOME”修改了，解决了paddlehub方面的问题。
但在通过paddlehub下载“ernie-tiny”模型时，一些“ernie-tiny”模型还是下载到.paddlenlp/models下
因此，我想知到如何修改.paddlenlp的路径，比如说放在D盘下。

系统：Windows10
OSError: Not found: "C:\Users\用户名\\.paddlenlp\models\ernie-tiny\spm_cased_simp_sampled.model": No such file or directory Error #2

**Comment by ZeyuChen** [comment, marked as answer]
@JoeLiaoCHN 
Linux下， export PPNLP_HOME=“其他非中文的目录即可"
Windows下配置环境变量 PPNLP_HOME 到其他非中文目录，重启即可

**Reply by JoeLiaoCHN** [reply]
问题已解决，谢谢！
